docs/faq/models.txt

```diff
@@ -69,12 +69,11 @@ Does Django support NoSQL databases?
 
 NoSQL databases are not officially supported by Django itself. There are,
 however, a number of side project and forks which allow NoSQL functionality in
```

Comment:
While you're here: `project` → `projects`.

Reply:
Changed.